src/main.py

```diff
@@ -11,17 +11,69 @@
 
 def welcome(update, context):
     for new_user_obj in update.message.new_chat_members:
-        user = str(new_user_obj['first_name']) + " " + str(new_user_obj['last_name'])
+        if( new_user_obj['last_name'] is None):
+            user = str(new_user_obj['first_name'])
+        else:
+            user ="(" + str(new_user_obj['first_name']) + " " + str(new_user_obj['last_name']) + ")"
+
+        if( new_user_obj['username'] is None):
+            username = ""
+        else:
+            username = "@" + str(new_user_obj['username']) + " "
```

Comment:
We can move this to a separate function, namely `get_name`. It would make our code more modular, easy to read and test.

Reply:
Then, please create a unit test for this function. You can refer here: https://docs.pytest.org/en/6.2.x/